### PR TITLE
Inform Teams of winner

### PR DESCRIPTION
**Pull request by Blokkinen**
PT Story: https://www.pivotaltracker.com/story/show/132561829

Changes proposed in this pull request:

- Push to Question page list of scores and winning team

What I have learned working on this feature: 

- How to use Action Cable
- How to create a job
- Displays the team with the highest points first

Screenshots: 
skarmavbild 2016-10-24 kl 22 23 58

Ready for review!

### CODE REVIEW

File: app/views/partials/_winner.html.haml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.results
+  - data[:list].each do |team|
+    %p #{team[:team].name} got #{team[:score]} points
```
Comment (AmberWilkie) [inline app/views/partials/_winner.html.haml]: I would love to see a reverse sort like we talked about - so the highest scores would be at the top.

File: db/seeds.rb [hunk 2]
```diff
@@ -24,3 +24,9 @@
 Question.create(body: question3, answer: answer3, quiz: quiz2)
 Question.create(body: question4, answer: answer4, quiz: quiz2)
 Question.create(body: question5, answer: answer5, quiz: quiz2)
+
+Team.create(name: 'team1', quiz: quiz1)
+Team.create(name: 'team2', quiz: quiz1)
+Team.create(name: 'team3', quiz: quiz1)
+Team.create(name: 'team4', quiz: quiz1)
+Team.create(name: 'team5', quiz: quiz1)
```
Comment (tochman) [inline db/seeds.rb]: Why are we seeding this?

File: features/quizmaster/view_results_page.feature [hunk 3]
```diff
@@ -10,10 +10,10 @@ Background:
...
 
-Scenario: I view and send results
+Scenario: I view and send results 
   And I am on the quizmaster page for "Trivia"
   When I click the "Results" button
   Then I should see "10"
   When "Amber Rocks!" is looking at the quiz page for "Trivia"
   And I click the "Send Results" button
-  Then "Amber Rocks!" should see "Amber Rocks! won!"
+  Then "Amber Rocks!" should see "Amber Rocks! got 11 points"
```
Comment (tochman) [inline features/quizmaster/view_results_page.feature]: And nothing else? What about the rest of the teams results?

File: db/seeds.rb (hunk 2 above)
Comment (Blokkinen) [inline db/seeds.rb]: So that Viktoria could see more than one team in the list, to make sure we're listing it right.. Want me to remove it?

**Pull request merged**